docs/source/benchmarking.md

```diff
@@ -0,0 +1,69 @@
+# Benchmarking Performance
+
+Benchmarking packet parsing is challenging because performance is greatly impacted by the complexity of the packet 
+structures being parsed. There are a few measures by which we can assess the performance of Space Packet Parser.
+
+*Note: Throughout the Space Packet Parser repo and documentation space, 
+B/kB means bytes/kilobytes and b/kb means bits/kilobits.*
+
+Common factors affecting performance:
+- Presence of calibrators and context calibrators
+- Complexity of container inheritance structure
+- Number and size of fields in a packet
+- Presence of large binary blobs (=high kbps)
+
+## Packets Per Second
+
+This is a metric we are often asked about. Unfortunately, the answer is that it depends on which packets are 
+being parsed: how many fields are in each packet and how much extra work the parser is doing to sort out complex 
+packet structures and evaluate calibrators.
+
+## Kilobits Per Second
+
+This metric is often used when discussing data volumes and downlink bandwidths to make sure that a data processing 
+system can keep up with the data rate from a spacecraft in the time allowed for processing. This number is also 
+affected by packet structures. It will be high for simple packets containing large binary blobs and low for 
+complex packets containing many small fields.
+
+## Results 
+
+As a baseline, for relatively simple packets (these are JPSS-1 spacecraft geolocation packets containing attitude 
```

Comment:
Add in where you ran this test. On a local M3 mac or whatever it is to give people a quick estimate of your processing power.